### ISSUE TITLE
Prevents Vanilla Mobs from Spawning

### ISSUE THREAD

**Issue by Riffmeister** [Lycanite/LycanitesMobs#195]
I have played with the configs, and have tried setting the lycanites mob spawn to false, but still no vanilla mobs spawn. Then after removing the mod completely I was able to get vanilla mobs (zombies, skeletons, etc.) to spawn. Is this a known issue or is there a workaround? Also, I do have mo'creatures installed, but it seems strange that I can get vanillas to spawn just by removing lycanites? Really awesome mod.

**Comment by Lycanite** [comment]
Lycanites Mobs does adjust where some vanilla mobs spawn, for example Skeletons don't spawn in Deserts as Clinks are kind of their replacement. However you can turn this off in each of the general configs such as desertmobs-general.cfg, in there is a setting called something like Control Vanilla Spawns, search for something like that and set it to false. Let me know if you're still having problems. :D

**Comment by Lycanite** [comment]
Happy New Year! :D
I'd try testing it on a new world first to make sure no other factors are in the way, if you're still getting problems could you send me a zip of your lycanites mobs configs and I'll take a look for ya. Also is this 1.7.10, 1.9 or 1.10?

**Comment by xlxAciDxlx** [comment]
@Riffmeister If you're still having trouble, try running `/kill @e[EntityHostile]` and exploring afterwords. Keep in mind the `EntityHostile` bit may need to be changed, perhaps Lycanite can offer a better suggestion. Pretty sure that would only target hostile mobs though, shouldn't kill anything important.

At least in my modpack I've had to run it once, and after running it just once the spawn rate seemed to be more "natrual"

**Comment by Riffmeister** [comment]
I found out what the problem was! I changed all of the lycanite configs to force despawning. Otherwise mobs, such as joust in my case, would take up the mob limit preventing other mobs from spawning. Might be something worth noting in the mod somewhere because it wasn't very intuitive? Let me know if you think this might cause further issues with other things, but so far it seems like it has been running pretty smoothly now 👍 Thank you guys for your advice and help and @Lycanite thank you for a very full and immersive mod. Very fun to play with. Also, I have been playing on 1.10.
